conans/client/manager.py

```diff
@@ -63,76 +64,122 @@ def install(self, reference, install_folder, graph_info, remote_name=None, build
         written
         @param no_imports: Install specified packages but avoid running imports
         @param inject_require: Reference to add as a requirement to the conanfile
+        @param editable: Path to conanfile in the working copy to be linked as editable package
         """
 
+        # Ensure some consistency for editable packages
+        if editable:
```

Comment:
This doesn't belong here, it is too deep in the install process, while the "editable" should only reach to create the link in the conan-cache, no more.